### PR TITLE
Adds forensics gloves

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Sprite from Bay. Thanks, Bay. Bay is bae, as they say. *cough*

These leave no prints or material traces on anything, giving you that slight "it's not MY black gloves" edge during detective work.

Useful during both crime scene investigation and crime scene creation.

Mostly there for the cool factor. 

Traitors and such can get the objective to steal these. 

They still won't protect you from electric shock.

:cl: FlattyPatty
add: Added forensics gloves for the Detective.
/:cl:

### CODE REVIEW

**Comment by scrubmcnoob** [comment]
> Bay is bae, as they say

fraternizing with the enemy now @FlattestGuitar ?

**Comment by Anoonki** [comment]
An alternate solution to the "Well it's not MY black gloves" thing, is that when you label gloves, their fabric traces include the label, for whatever silly reason.

File: code/game/gamemodes/steal_items.dm [hunk 1]
```diff
@@ -132,6 +132,11 @@ datum/theft_objective/ai/check_special_completion(var/obj/item/device/aicard/C)
 	typepath = /obj/item/clothing/gloves/color/black/krav_maga/sec
 	protected_jobs = list("Head Of Security", "Warden")
 
+/datum/theft_objective/forensics_gloves
```
Comment (Fox-McCloud) [inline code/game/gamemodes/steal_items.dm]: These aren't worth polluting the theft objective table for.

**Comment by FalseIncarnate** [comment]
Wondering if these should be black or white gloves... White being reference to kid gloves of course, meant for ensuring you leave no traces.

**Comment by KasparoVy** [comment]
How does this look on Humans/Vox/Drask? Does it share the black gloves' on_mob sprite?

**Comment by FlattestGuitar** [comment]
@KasparoVy yep. Looks just like combat or black gloves.

File: code/game/gamemodes/steal_items.dm (hunk 1 above)
Comment (Vivalas) [inline code/game/gamemodes/steal_items.dm]: I don't see why not, seems experimental and pricey.

**Comment by Vivalas** [comment]
Hehe, now there is actually a reason to screech at officers for touching everything at a crime scene. :+1:

File: code/game/gamemodes/steal_items.dm (hunk 1 above)
Comment (taukausanake) [inline code/game/gamemodes/steal_items.dm]: Doesn't really seem like either of those to be honest. Just rare as they are the only ones on the station. I think they should be fine for theft, how many things do you have on that table anyways?

**Comment by Fox-McCloud** [comment]
These are just gloves; please remove them from the theft table; not every job specific thing has to be a theft objective.

**Pull request merged**